app/notifications/user_update_notification.rb

```diff
@@ -0,0 +1,36 @@
+class UserUpdateNotification < Noticed::Base
+
+  deliver_by :database, debug: true
+
+
+  deliver_by :database, if: :database_notifications?
+  deliver_by :email, mailer: "UserMailer", method: "update_user", if: :email_notifications?
+
+  # Variables that tells the system if it should send a notification if the recipient isn't subscribed
+  DEFAULT_SEND_NOTIFICATION = true
+  DEFAULT_SEND_EMAIL_NOTIFICATION = true
+
+  # Add required params
+  param :event_user
+
+  def message
+    "The user #{params[:event_user].to_s} - #{params[:event_user].email} has been updated."
```

Comment:
```suggestion
    "The user #{params[:event_user]} - #{params[:event_user].email} has been updated."
```
No need to call `to_s` explicitly